doc/man7/EVP_KEM-EC.pod

```diff
@@ -66,6 +66,10 @@ L<EVP_KEYMGMT(3)>,
 L<EVP_PKEY(3)>,
 L<provider-keymgmt(7)>
 
+=head1 HISTORY
+
+This functionality was added in OpenSSL 3.2.
+
```

Comment:
It would be nice to split this out into a separate commit so we might be able to cherry-pick to 3.1/3.0?

Reply:
Done- separate PR created,